### PR TITLE
October2017

### LINKED ISSUE #18: Create file for me - @YashSV
- [x] Create a Branch
- [x] Add Commits
- [x] Open a Pull request
- [ ] Collaborate
- [ ] Receive Approval :+1: 
- [ ] Merge :shipit:

### PR DESCRIPTION
**Pull request by YashSV**
closes #18

### CODE REVIEW

File: YashSV-Mysore.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+### Mysore, India
+
+#### Places to go
+
+#### | Places     | State     |
+| :------------- | :------------- |
+| Mysore       | 1      |
+
+#### Places to Eat
+
```
Comment (chrisatkins) [inline YashSV-Mysore.md:13]: I'm hungry!

**Comment by chrisatkins** [comment]
I've heard of the Taj Mahal!

**Comment by YashSV** [comment]
@chrisatkins : Yes, its a nice place :+1:

File: YashSV-Mysore.md [hunk 2]
```diff
@@ -6,6 +6,7 @@
 | :------------- | :------------- |
 | Mysore       | 1      |
 | Bangalore    | 2      |
+| Mangalore    |3       |
```
Comment (YashSV) [inline YashSV-Mysore.md]: Add new place name -1

**Comment by YashSV** [comment]
ready to :shipit: ?

**Comment by baidosters00d** [comment]
:shipit:

**Pull request merged**